CustomerApp/CustomerApp/Pages/YourOrderPage.xaml

```diff
@@ -0,0 +1,139 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             x:Class="CustomerApp.Pages.YourOrderPage"
+             BackgroundImageSource="YourOrderPage.png">
+    <ContentPage.Content>
+        <Grid
+            x:Name="yourOrderLayout">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="50" />
+                <RowDefinition Height="400" />
+                <RowDefinition Height="50" />
+                <RowDefinition Height="180" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="360"/>
+            </Grid.ColumnDefinitions>
+            
+            <!--Scroll view to display order items-->
```

Comment:
Great job on figuring all of this out and I really like your coding style and how much you actually got done! You've learned really quickly and have done a fantastic job with these changes!